multiscale_spatial_image/multiscale_spatial_image.py

```diff
@@ -24,17 +27,16 @@ class MultiscaleSpatialImage(DataTree):
       scale2
     """
 
-    def __init__(
+    def __init__(self, xarray_obj: DataTree):
+        self._obj = xarray_obj
```

Comment:
Perhaps something more specific than `_obj`, `_dt`, etc.

Reply:
changed to `_dt`